.github/workflows/build-images.yaml

```diff
@@ -30,6 +30,14 @@ jobs:
         id: add-branch-tag
         run: |
           echo "IMG_TAGS=${GITHUB_REF_NAME/\//-} ${{ env.IMG_TAGS }}" >> $GITHUB_ENV
+      - name: Set Authorino version
+        id: authorino-version
+        run: |
+          if [[ ${GITHUB_REF_NAME/\//-} =~ ^v[0-9]+\.[0-9]+\.[0-9]+(-.+)?$ ]]; then
```

Comment:
one minor thing regarding the semantic versioning... prefixing it with a `v` it's a common way to indicate that it's a version and it's also widely used... however, semantic versioning would mean without it. Also, in the [Makefile](https://github.com/Kuadrant/authorino/pull/345/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R121)  the regex is different, matching without the leading `v` and adding it afterwards... should we follow the same here?

Reply:
It’s the difference between the version number and the image tag.